RockPaperScissors/RockPaperScissors/Turn.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Turn.swift
+//  RockPaperScissors
+//
+//  Created by kiwi,bard on 2022/04/20.
+//
+
+enum Turn {
+    case computerTurn
+    case userTurn
+}
+
+extension Turn {
+    var turnResult: String {
+        switch self {
+        case .computerTurn:
+            return "컴퓨터"
+        case .userTurn:
+            return "사용자"
+        }
+    }
+}
```

Comment:
enum으로 Turn을 관리하시는군요? ㅎㅎ 너무 좋습니다
추가로 rawValue가 불편해서 turnResult 라는 변수를 만들어서 turn.turnResult로 사용하시는건 정말 좋은거 같아요!!

다만 약간의 코맨트를 드릴 수 있을것 같습니다
trun이라는 이름이 너무 중복되지 않나요?

```suggestion
enum Turn {
    case computer
    case user
}

extension Turn {
    var message: String {
        switch self {
        case .computer:
            return "컴퓨터"
        case .user:
            return "사용자"
        }
    }
}
```

Reply:
`turn` 생략하였습니다